src/components/Person/Person.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  function partner(s) {
+    return s === 'm' ? 'wife' : 'husband';
+  }
```

Comment:
i dont think you need a function for this. a simple variable and a ternary should be enough